chainer/functions/connection/convolution_2d.py

```diff
@@ -308,58 +308,91 @@ def convolution_2d(x, W, b=None, stride=1, pad=0, use_cudnn=True,
     - :math:`n` is the batch size.
     - :math:`c_I` and :math:`c_O` are the number of the input and output
       channels, respectively.
-    - :math:`h` and :math:`w` are the height and width of the input image,
+    - :math:`h_I` and :math:`w_I` are the height and width of the input image,
       respectively.
-    - :math:`k_H` and :math:`k_W` are the height and width of the filters,
+    - :math:`h_K` and :math:`w_K` are the height and width of the filters,
       respectively.
+    - :math:`h_P` and :math:`w_P` are the height and width of the spatial
+      padding size, respectively.
 
-    Args:
-        x (~chainer.Variable): Input variable of shape :math:`(n, c_I, h, w)`.
-        W (~chainer.Variable): Weight variable of shape
-            :math:`(c_O, c_I, k_H, k_W)`.
-        b (~chainer.Variable): Bias variable of length :math:`c_O` (optional).
-        stride (int or pair of ints): Stride of filter applications.
-            ``stride=s`` and ``stride=(s, s)`` are equivalent.
-        pad (int or pair of ints): Spatial padding width for input arrays.
-            ``pad=p`` and ``pad=(p, p)`` are equivalent.
-        use_cudnn (bool): If ``True``, then this function uses cuDNN if
-            available.
-        cover_all (bool): If ``True``, all spatial locations are convoluted
-            into some output pixels. It may make the output size larger.
-        deterministic (bool): The output of this function can be
-            non-deterministic when it uses cuDNN.
-            If this option is ``True``, then it forces cuDNN to use
-            a deterministic algorithm. This option is only available for
-            cuDNN version >= v3.
-
-
-    Returns:
-        ~chainer.Variable: Output variable.
-
-    The two-dimensional convolution function is defined as follows.
     Then the ``Convolution2D`` function computes correlations between filters
-    and patches of size :math:`(k_H, k_W)` in ``x``.
+    and patches of size :math:`(h_K, w_K)` in ``x``.
     Note that correlation here is equivalent to the inner product between
     expanded vectors.
     Patches are extracted at positions shifted by multiples of ``stride`` from
-    the first position ``-pad`` for each spatial axis.
+    the first position ``(-h_P, -w_P)`` for each spatial axis.
     The right-most (or bottom-most) patches do not run over the padded spatial
     size.
 
-    Let :math:`(s_Y, s_X)` be the stride of filter application, and
-    :math:`(p_H, p_W)` the spatial padding size. Then, the output size
-    :math:`(h_O, w_O)` is determined by the following equations:
+    Let :math:`(s_Y, s_X)` be the stride of filter application. Then, the
+    output size :math:`(h_O, w_O)` is determined by the following equations:
 
     .. math::
 
-       h_O &= (h + 2p_H - k_H) / s_Y + 1,\\\\
-       w_O &= (w + 2p_W - k_W) / s_X + 1.
+       h_O &= (h_I + 2h_P - h_K) / s_Y + 1,\\\\
+       w_O &= (w_I + 2w_P - w_K) / s_X + 1.
```

Comment:
Could you write the formula when `cover_all` is `True`, too, as it changes the output size.